src/netex-convertor/netexGenerator.ts

```diff
@@ -174,16 +180,39 @@ const netexGenerator = (
     };
 
     const updateServiceFrame = (serviceFrame: NetexObject): NetexObject | null => {
+        const serviceFrameToUpdate = { ...serviceFrame };
         if (isMultiServiceTicket(ticket)) {
-            const serviceFrameToUpdate = { ...serviceFrame };
             serviceFrameToUpdate.id = `epd:UK:${ticket.nocCode}:ServiceFrame_UK_PI_NETWORK:${coreData.placeholderGroupOfProductsName}:op`;
-
             serviceFrameToUpdate.lines.Line = getLinesList(ticket, coreData.website, operatorData);
 
             return serviceFrameToUpdate;
-        }
+        } else if (isPointToPointTicket(ticket)) {
+            serviceFrameToUpdate.id = `epd:UK:${ticket.nocCode}:ServiceFrame_UK_PI_NETWORK:${coreData.lineIdName}:op`;
+            serviceFrameToUpdate.lines.Line.id = coreData.lineName;
+            serviceFrameToUpdate.lines.Line.Name.$t = coreData.operatorPublicNameLineNameFormat;
+            serviceFrameToUpdate.lines.Line.PublicCode.$t = coreData.lineName;
+            serviceFrameToUpdate.lines.Line.PrivateCode.$t = coreData.nocCodeLineNameFormat;
+            serviceFrameToUpdate.lines.Line.OperatorRef.ref = coreData.nocCodeFormat;
+            serviceFrameToUpdate.lines.Line.OperatorRef.$t = coreData.opIdNocFormat;
+            serviceFrameToUpdate.lines.Line.Description.$t = ticket.serviceDescription;
+
+            if (isReturnTicket(ticket)) {
+                const outboundStops = getPointToPointScheduledStopPointsList(ticket.outboundFareZones);
+                const inboundStops = getPointToPointScheduledStopPointsList(ticket.inboundFareZones);
+                const scheduledStopPointList: ScheduledStopPoints[] = outboundStops.concat(inboundStops);
+                serviceFrameToUpdate.scheduledStopPoints.ScheduledStopPoint = [
+                    ...new Set(scheduledStopPointList.map(({ id }) => id)),
+                ].map(e => scheduledStopPointList.find(({ id }) => id === e));
+            } else {
+                serviceFrameToUpdate.scheduledStopPoints.ScheduledStopPoint = getPointToPointScheduledStopPointsList(
+                    ticket.fareZones,
+                );
+            }
 
-        return null;
+            return serviceFrameToUpdate;
+        } else {
+            return null;
```

Comment:
This return can be outside of the else since the other ifs all return, then we can remove the eslint disable line

Reply:
I can move the return outside of the final if, but I can't remove the eslint disable because it bizarrely complains about it being un-needed, when it really clearly is needed above.